addons/website_event/controllers/main.py

```diff
@@ -389,6 +389,17 @@ def registration_confirm(self, event, **post):
         registrations = self._process_attendees_form(event, post)
         attendees_sudo = self._create_attendees_from_registration_post(event, registrations)
 
+        return request.redirect('/event/%s/registration/success?registration_ids=%s' % (event.id, ",".join(map(str, attendees_sudo.ids))))
```

Comment:
Ok I see why you use map, it won't join integers.

Then maybe something like `",".join([str(id) for id in attendees_sudo.ids])` ?
I would like to avoid the "map" usage and use a simple list comprehension instead.

Also, did you fix the comme message? It looks unchanged (see my previous review message).

Reply:
Changes Done!